index.js

```diff
@@ -1,14 +1,14 @@
 const { random } = require('faker');
 
-function createBadAcronym(seedWord) {
+function createBadAcronym(seedWord = 'loser') {
     return seedWord.split('').map(letter => {
-        const newWord = random.word();
-        wordArray = newWord.split('');
-        wordArray[0] = letter;
-        return wordArray.join('');
+        let word = ' ';
+        while (word[0] != letter) {
```

Comment:
I'm really worried about performance here. This run too well, can we make it clunkier?